src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -39,35 +40,35 @@
     /**
      * Amount of xp the builder gains each building (Will increase by attribute modifiers additionally)
      */
-    private static final double XP_EACH_BUILDING = 2.5;
+    private static final double   XP_EACH_BUILDING              = 2.5;
     /**
      * How often should intelligence factor into the builders skill modifier.
      */
-    private static final int INTELLIGENCE_MULTIPLIER = 2;
+    private static final int      INTELLIGENCE_MULTIPLIER       = 2;
     /**
      * How often should strength factor into the builders skill modifier.
      */
-    private static final int STRENGTH_MULTIPLIER = 1;
-    /**
-     * Position where the Builders constructs from.
-     */
-    private BlockPos workFrom = null;
+    private static final int      STRENGTH_MULTIPLIER           = 1;
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)